sem_covid/entrypoints/streamlit_visualizers/semantic_similarity.py

```diff
@@ -20,7 +20,7 @@
 import plotly.express as px
 
 DATE_COLUMN_NAME = 'date'
-EMBEDDING_COLUMN_NAME = 'document_embeddings'
+EMBEDDING_COLUMN_NAME = 'document_embeddings_use'
```

Comment:
We shall pay attention that this may have unexpected consequences.

Also, in the function time_range_selector(...,...) I would not hardcode any values there, especially MAX value, which shall be "NOW". So the min value can be a module Constant, while the max shall be dynamically generated based on the present date.